boards/posix/native_posix/timer_model.c

```diff
@@ -22,14 +22,14 @@
 #include <time.h>
 #include <stdbool.h>
 #include <math.h>
+#include <kernel.h>
```

Comment:
same comment here and in a few other files

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/20119#discussion_r341810118